frontend/src/app/upload/page.tsx

```diff
@@ -65,14 +65,18 @@ export default function Home() {
   }, [clipId]);
 
   const checkProgress = async () => {
-    const rest = await fetch(`/api/clips/${clipId}/progress`);
+    const rest = await fetch(`/api/clips/progress?cid=${clipId}`);
```

Comment:
We should move these to functions in the future, this can be disregarded for now though once we're working out the API

Reply:
Agree